Maple2.Server.Game/GameServer.cs

```diff
@@ -18,14 +19,22 @@ public class GameServer : Server<GameSession> {
     private readonly FieldManager.Factory fieldFactory;
     private readonly HashSet<GameSession> connectingSessions;
     private readonly Dictionary<long, GameSession> sessions;
+    private readonly Dictionary<string, GameEvent> eventCache = new();
 
     public int Channel => Target.GameChannel;
+    
+    #region Autofac Autowired
```

Comment:
You can't do this unless you have `PropertiesAutowired()` which is why you have `context.Resolve<GameStorage>()`. Just define this normally to avoid confusion:

```cs
private GameStorage gameStorage;
```

Also you shouldn't need to use `context.Resolve<GameStorage>()`, just add `GameStorage gameStorage` as a parameter to the constructor.